docs/application/vscode-ext/web.md

```diff
@@ -42,7 +42,7 @@ If you do not have a real device, you can run your application in a virtual envi
 
 - To use Tizen Package Manager, follow these steps:
   1. In VS Code, click **View** > **Command Palette**.
-  2. In the input field of the **Command Palette** that appears, type **Run Package-Manager** and press **Enter**.
+  2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Package-Manager** and press **Enter**.
```

Comment:
In line number 9, please change "check" to "see":

Before installing VS Code Extension for Tizen Web, see [Prerequisites of Visual Studio Code Extension for Tizen](index.md).

Reply:
done.

Reply:
done.